pkg/volume/util.go

```diff
@@ -47,6 +50,21 @@ func GetAccessModesAsString(modes []api.PersistentVolumeAccessMode) string {
 	return modesAsString
 }
 
+func GetAccessModesFromString(modes string) []api.PersistentVolumeAccessMode {
```

Comment:
This is a good suggestion.  It ended with callers of these func having fewer imports.  Less is less.